src/main/webapp/test/mashup.jsp

```diff
@@ -8,429 +8,519 @@
-
-                <div class="pageinfo">Terms Of Use Page</div>
-                <div id="terms">
-                    <iframe class="full-width" src="/terms.jsp"></iframe>
-                </div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Request Account Page</div>
-                <div id="request">
-                    <iframe class="full-width" src="/request.jsp"></iframe>
-                </div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Usermap Page</div>
-                <div id="usermap">
-                    <iframe class="full-width" src="/usermap.jsp"></iframe>
-                </div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Student Help Page</div>
-                <div id="studentHelp">
-                    <iframe class="full-width" src="/studentHelp.jsp"></iframe>
-                </div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Instructor Help Page</div>
-                <div id="instructorHelp">
-                    <iframe class="full-width" src="/instructorHelp.jsp"></iframe>
-                </div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Deadline Exceeded Error Page</div>
-                <div id="deadlineExceededErrorPage"></div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Error Page</div>
-                <div id="errorPage"></div>
-                <br><hr class="hr-bold"><br>
-
-                <div class="pageinfo">Entity Not Found Page</div>
-                <div id="entityNotFoundPage"></div>
-                <br><hr class="hr-bold"><br>
-                <br></br>
-                <br></br>
-                <br></br>
-                <br></br>
-            </div>
+  <head>
+    <meta http-equiv="X-UA-Compatible" content="IE=8">
+    <meta content="width=device-width, initial-scale=1.0" name="viewport">
+
+    <title>TEAMMATES - Webpage Compilation</title>
+
+    <link rel="shortcut icon" href="/favicon.png">
+    <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_CSS %>">
+    <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>">
+    <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet">
+    <style>
+      .full-width {
+        width: 100%;
+      }
+
+      .pageinfo {
+        font-size: 14px;
+        color: red;
+      }
+
+      .hr-bold {
+        border: 2px red solid;
+      }
+    </style>
+    <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY %>"></script>
+    <script type="text/javascript" src="<%= FrontEndLibrary.BOOTSTRAP %>"></script>
+    <script type="text/javascript" src="/js/common.js"></script>
+  </head>
+  <body>
+    <div class="container theme-showcase">
+      <div class="row">
+        <div id="dhtmltooltip">
+        <div class="alert alert-warning">
```

Comment:
Looks like `<div id="dhtmltooltip">` is unclosed. A [search](https://github.com/TEAMMATES/teammates/search?utf8=%E2%9C%93&q=dhtmltooltip&type=) in the repo shows that `commonAssertRegexTestPage.html` uses it like this: `<div id="dhtmltooltip"></div>`.